microservices/external_data_query/external_data_query/pubmed/api_models.py

```diff
@@ -3,7 +3,7 @@
 import dataclasses
 from typing import List, Optional
 
-from pubmed_query.mixins import BaseModelMixin
+from external_data_query.mixins import BaseModelMixin
 
 @dataclasses.dataclass
```

Comment:
Please use Pydantic dataclasses everywhere instead of standard dataclasses module. FastAPI is built on top of Pydantic